### PR TITLE
add acks test

### PR DESCRIPTION
**Pull request by evanmcc**
- bugfix for ack table name creation
- no need to keep an acks table on the tail
- add a test that ensures that we're actually getting the acks in a timely fashion

### CODE REVIEW

**Comment by evanmcc** [comment]
closes #21

**Comment by evanmcc** [comment]
I guess it doesn't close it so much as checks that it was already happening.

**Comment by tsloughter** [comment]
So this is ready to review now?

File: src/vg_active_segment.erl [hunk 1]
```diff
@@ -168,16 +169,21 @@ code_change(_, State, _) ->
...
                                                               byte_count=ByteCount}) ->
                                      {NextId, Size, Bytes} = vg_protocol:encode_log(Id, Record),
                                      StateAcc1 = #state{pos=Position1} = maybe_roll(Size, StateAcc),
                                      update_log(Bytes, StateAcc1),
                                      StateAcc2 = StateAcc1#state{byte_count=ByteCount+Size},
                                      StateAcc3 = update_index(StateAcc2),
-
-                                     vg_pending_writes:add(Tab, Id, Bytes),
+                                     case Next of
+                                         %%solo -> ok;
```
Comment (tsloughter) [inline src/vg_active_segment.erl:181]: should just delete this

**Review by tsloughter — APPROVED** [review]

**Pull request merged**